posts/2021-04-12-async-vision-doc-shiny-future.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "Brainstorming Async Rust's Shiny Future"
+author: Niko Matsakis
+description: "Brainstorming Async Rust's Shiny Future"
+team: the Async Foundations Working Group <https://rust-lang.github.io/wg-async-foundations/>
+---
+
+On March 18th, we [announced the start of the Async Vision Doc][announce] process. Since then, we've landed [17 "status quo" stories][sq] and we have [5 more stories in open PRs][prs]; [Ryan Levick] and [I] have also hosted more than ten [collaborative][] [writing][] [sessions][] over the course of the last few weeks.
```

Comment:
total nit but i think you can omit the trailing [], don't think it makes a diff either way tho

Reply:
You can't here because `[collaborative] [writing]` interprets the [writing] as the name of the link. I hit this trap a few times...

Reply:
oh whoa, right. TIL! markdown is so fun hahahaha.